docs/Vue.js/StateManagement.md

```diff
@@ -0,0 +1,126 @@
+---
+id: Vue.js state management
```

Comment:
change `id` in good format because we are using `id` for unique and for routing like `url` so, write in right way for SEO and best practice